src/NumericalIntegration.jl

```diff
@@ -167,7 +167,9 @@ function integrate(x::AbstractVector, y::AbstractVector, m::RombergEven)
             rombaux[j, currcol] = (n_k*rombaux[j-1, currcol] - rombaux[j-1, prevcol])/(n_k - 1)
         end
 
-        if i > maxsteps//3 && norm(rombaux[i, prevcol] - rombaux[i+1, currcol], Inf) < m.acc
+        # Clunky way to address Unitful compatibility, while also allowing for vectors
+        normval = norm(rombaux[i, prevcol] - rombaux[i+1, currcol], Inf)
+        if i > maxsteps//3 && normval/oneunit(normval) < m.acc
```

Comment:
One could precompute `div(maxsteps, 3)` and `m.acc * oneunit(v)` outside of the `for` loop to avoid the divisions here in every step, I guess.

Reply:
Nice, I've included those.